src/main/java/io/github/thebusybiscuit/slimefun4/api/events/TalismanActivatedEvent.java

```diff
@@ -0,0 +1,88 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.magical.talismans.Talisman;
+
+/**
+ * 
+ * This {@link PlayerEvent} is called when a {@link Player} activates a {@link Talisman}
+ * 
+ * @author cworldstar
+ *
+ */
+
+public class TalismanActivatedEvent extends PlayerEvent implements Cancellable {
+
+	private static HandlerList Handlers = new HandlerList();
+	private Talisman Talisman;
+	private ItemStack TalismanItem;

```

Comment:
In Java fields are `camelCased`.

```suggestion
	private Talisman talisman;
	private ItemStack talismanItemStack;

```